etc/kayobe/kolla/globals.yml

```diff
@@ -69,5 +69,131 @@ prometheus_ceph_mgr_exporter_endpoints:
 {% endfor %}
 {% endif %}
 
-#############################################################################
+
+########################################
```

Comment:
Definitely too big for Yoga. Merge conflict in Antelope will be annoying but possibly worth it. Second merge conflict in Caracal when if gets moved into k-a will also be annoying :)

Reply:
One way around this could be to use a separate group_vars file in etc/kayobe/kolla/inventory/group_vars/prometheus-blackbox-exporter to define the rules.

Reply:
Moved to a new file and rebased on antelope